### PR TITLE
topology: Add Waves codec to mt8186 topology

### PR DESCRIPTION
**Pull request by barry-waves**
config: Add Waves override config for mt8186_chrome.
module_adapter: Enable module_adapter and Waves codec.

Signed-off-by: barry.jan <barry.jan@waves.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Review by kuanhsuncheng — APPROVED** [review]

**Comment by lgirdwood** [comment]
test this please

**Comment by lgirdwood** [comment]
I've started the CI.

File: src/arch/xtensa/configs/override/mt8186_chrome_waves.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CONFIG_CAVS_LPS=y
```
Comment (yaochunhung) [inline src/arch/xtensa/configs/override/mt8186_chrome_waves.config:1]: Why need this for mt8186? It seems used to enable Intel cAVS Low Power Sequencer

**Pull request closed without merging**